nilmtk/dataset_converters/caxe/convert_caxe.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pandas as pd
+import numpy as np
+from nilmtk.datastore import Key
+from nilmtk.utils import check_directory_exists, get_datastore, get_module_directory
+from nilm_metadata import convert_yaml_to_hdf5
+
+def convert_caxe(file_path):
+    '''
+    Parameters
+    ------------
+    Takes input csv_file name to be tested as string.
+    Data columns of the csv should contain following the following values in columns:
+    timestamp,reactive_power,apparent_power,current,frequency,voltage,active_power) 
+    Converts it into hdf5 Format and save as test.h5.
+    '''
+    df = pd.read_csv(f'{file_path}',names =['timestamp','R','A','C','F','V','T'])
+    column_mapping = {
+        'F': ('frequency', ""),
+        'V': ('voltage', ""),
+        'T': ('power', 'active'),
+        'C': ('current', ''),
+        'R': ('power', 'reactive'),
+        'A': ('power', 'apparent'),
+    }
+
+    TIMESTAMP_COLUMN_NAME = "timestamp"
```

Comment:
Why putting the global variables *inside* the function? They are not global in that case.